tests/phpunit/lib/searchable.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+/**
+ * Unit tests for Searchable trait
+ * @package lib/tests
+ */
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests for the Searchable trait
+ */
+class Hm_Test_Searchable extends TestCase {
+
+    public function setUp(): void {
+        if (!defined('APP_PATH')) {
+            require_once __DIR__.'/../bootstrap.php';
```

Comment:
Proper bootstrap means we don't call these inside the unit tests themselves. Why would the app path not be defined? Tests should be executed by a single phpunit command which should always properly bootstrap the framework.